apps/addons/models.py

```diff
@@ -624,12 +624,15 @@ def share_url(self):
 
     @property
     def automated_signing(self):
-        # We allow automated signing for add-ons which are not listed, and
-        # have not asked for side-loading privileges (full review).
+        # We allow automated signing for add-ons which are not listed.
         # Beta versions are a special case for listed add-ons, and are dealt
         # with on a file-by-file basis.
-        return not (self.is_listed or
-                    self.status in (amo.STATUS_NOMINATED, amo.STATUS_PUBLIC))
```

Comment:
A malicious or very opportunistic application installer could force extension installation in more than one way, so it doesn't make much of a difference if the signed extension has the sideloading flag or not. We required a higher review level for sideloaded extensions because it has historically been a more effective distribution mechanism of greyware, among other reasons. We will continue to pay more attention to those add-ons, but all reviews will happen post-signature.